app/views/appointment_summaries/generate.html.erb

```diff
@@ -0,0 +1,3 @@
+<div class="page-header">
+  <h1><%= title 'Generating PDF' %></h1>
```

Comment:
Might just be worth adding a quick explainer?

`<p>The summary document will open in a new window.</p>`